p2p/host.go

```diff
@@ -443,17 +444,9 @@ func (host *HostV2) Close() error {
 
 // PeerConnectivity returns total number of known, connected and not connected peers.
 func (host *HostV2) PeerConnectivity() (int, int, int) {
-	connected, not := 0, 0
-	peers := host.h.Peerstore().Peers()
-	for _, peer := range peers {
-		result := host.h.Network().Connectedness(peer)
-		if result == libp2p_network.Connected {
-			connected++
-		} else if result == libp2p_network.NotConnected {
-			not++
-		}
-	}
-	return len(peers), connected, not
```

Comment:
For me old code looks safer approach. For each peer it checks Connectedness. If you are sure about 
```golang
connected := len(host.h.Network().Peers())
```
Then no issue.

Reply:
We have about 16000 knows peers, so it will make 16k checks. Only concern here, that it works as the previous one.

Reply:
It seems this function is only used for public API (```GetNodeMetadata```). So, if you have properly tested the output, it should be fine.